### DISCUSSION TITLE
Del.

### DISCUSSION THREAD

**Discussion by SunGreen777** [lllyasviel/stable-diffusion-webui-forge#2426] [General]
lllyasviel  It’s very inconvenient to switch between interfaces, I admit. I like the interfaces of A1111 and ForgeUI, but, like with A1111, there hasn’t been any news from you regarding further development for a long time. I see that you spent the past month working on IcLight, but what about ForgeUI? At the beginning of autumn, you made great progress and gave us hope that the A1111 interface would continue to thrive and even improve. However, there haven’t been any significant updates for quite a while. Do you plan to resume work? Please don’t ignore this. Writing a few lines about your plans isn’t too much to ask.

**Comment by ScionoicS** [comment]
Troll brigade is in full force on their like button here.  

lllyasviel isn't responsible to y'all. This concern trolling is obvious concern trolling.  

Calm down. You're not their boss. This is a voluntary project. 
 
@olegkokh @giovannagriffo @nektosu @bsdesigner @schlaus @amazingpacoo  All little hype trolls. They likely all come from the same discord server and mean to troll. Nothing more.  

Try offering something substantial one day.

**Reply by bsdesigner** [reply]
Give me at least one specific example of my trolling towards lllyasviel. I used his Fooocus product for a long time, and after the Flux model was released, I use Forge. If you don't provide examples of what you call “trolling”, we'll consider you a blabbermouth and a liar. 

P.S. Even though I am registered on Discord, I don't go there and I don't do any activity there. So your accusations are not only groundless, but also false. 
N.B. I suggest you choose one of two things - either apologize or shut up!

**Reply by Hujikuio** [reply]
what in the world

**Reply by Niiickel** [reply]
What's happening in this thread? OP just asked a question, yeah his wording was kinda weird, but he still just asked about future development and that's totally legit.

**Comment by JinJomes** [comment]
You should ask for a refund - soon will be the coming of spring, with much work to be done for the harvest.

**Comment by SunGreen777** [comment]
I created this thread.
What did I do wrong?
Did I ask about plans? Is that so bad?
Have you never heard of a roadmap?
At the beginning of autumn, lllyasviel was actively developing the project and suddenly stopped.
There have been no updates for two months.
lllyasviel is silent. Recently, we lost A1111. Of course, we want to hear some words of hope.
Now tell me, why am I being called a troll? I was polite and asked a legitimate question.
I titled the thread this way because I wanted to hear from the project's developer.
If you are in contact with lllyasviel, it would be more helpful to share info from them rather than calling everyone trolls.

**Reply by HMRMike** [reply]
You're just repeating a scenario that was before the Forge rewrite. 
The developer owes us nothing, really, not any single plan, no reason to "calm us down". Let him cook how he pleases.
We have this for info, make of that what you will
https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/main/NEWS.md

**Reply by SunGreen777** [reply]
> You're just repeating a scenario that was before the Forge rewrite. The developer owes us nothing, really, not any single plan, no reason to "calm us down". Let him cook how he pleases. We have this for info, make of that what you will https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/main/NEWS.md

Nobody demands anything; against the backdrop of the stopped A1111, silence provokes questions.
I've gone to Comfy for now.

**Reply by ScionoicS** [reply]
The A1111 project's dev branch was updated most recently 2 weeks ago. Nothing indicates that it has stopped other than your own bias diagnosis. It's all rooted in trolling these devs who offer their work for free while you do nothing. You don't get to declare it a dead project to anyone other than yourself. 

There are a lot of reasons why I called you a troll. Concern trolling is nothing new. You're being highly passive aggressive about it.

**Reply by nektosu** [reply]
>> @olegkokh @giovannagriffo @nektosu @bsdesigner @schlaus @amazingpacoo All little hype trolls.
> There are a lot of reasons why I called you a troll.

I would ask you to list the reasons why you insulted several people at once, including me.
But I think you're being aggressive.
You are ill-mannered and I don't expect you to apologize.
People like you should just be ignored.
You are not worthy of dialogue and attention. 
Goodbye.

**Reply by SunGreen777** [reply]
> There are a lot of reasons why I called you a troll. Concern trolling is nothing new. You're being highly passive aggressive about it.

Below I showed that lllyasviel is interesting to me, a screenshot of CivitAi. Or what was the purpose of my suggestion? 
You as a representative could have discussed this with CivitAi.
Show the posts you thought were trolling. 
***
how many updates in A1111?
![image](https://github.com/user-attachments/assets/257c69e2-9964-48ed-997e-65426285844f)
You're not a designer and probably not a Ui user. Otherwise you would understand how important it is to get used to the interface.
Jumping from one interface to another interferes with your work. 
If A1111 had said here is the interface, pay $20 there would be no questions.  
Any noteworthy product sooner or later becomes commercial. 
And to abandon the product without any explanation, leaving users in a suspended state is disrespectful to users and application developers.

**Reply by schlaus** [reply]
Dear all,

when a random person says something you do not like, don't get provoked. It does not matter. It's water under the bridge.

All is well. Have an awesome day <3

**Reply by SunGreen777** [reply]
> Dear all,
> 
> when a random person says something you do not like, don't get provoked. It does not matter. It's water under the bridge.
> 
> All is well. Have an awesome day <3

My post starts with a statement that I like Forge and ends with a question about future work.
Do you see provocation in a polite question? There is no provocation there, it seems to you, the provocation came from a moderator with unsubstantiated accusations in a thread he was not invited to.

**Reply by ScionoicS** [reply]
> > > @olegkokh @giovannagriffo @nektosu @bsdesigner @schlaus @amazingpacoo All little hype trolls.
> > > There are a lot of reasons why I called you a troll.
> 
> I would ask you to list the reasons why you insulted several people at once, including me. But I think you're being aggressive. You are ill-mannered and I don't expect you to apologize. People like you should just be ignored. You are not worthy of dialogue and attention. Goodbye.

I'm being very directly aggressive about it. I called you all out directly because you gave the big thumbs up to this troll post, trying to show your support while staying in the shadows. Do better.

**Reply by ScionoicS** [reply]
>   to abandon the product without any explanation, leaving users in a suspended state is disrespectful to users and application developers.

No one has abandoned this project and you're owed nothing. Go be a mini middle manager somewhere else.

**Comment by SunGreen777** [comment]
As for me, two months ago, I suggested that the CivitAi YouTube channel interview lllyasviel. 
Am I really a troll? 
You asked for suggestions, so take a look at my posts there are suggestions there.
Maybe I understand the potential of the project's creator and simply want the interface I like to keep evolving?
I want new people to join, create tutorials. But if you just abandon it and stay silent, people will leave.
![image](https://github.com/user-attachments/assets/bec16753-15ce-4f9f-92f6-130f13ea27ec)

**Comment by DIGITAL-SURFER** [comment]
I for one am very grateful for lllyasviel's contributions.  He took the completely unusable Automatic1111 and produced Forge, which allowed us mere mortals without the mega expensive 24GB cards, to be able to create our own imagery and develop our AI skills.

Illyasviel also produced the marvellous Fooocus which meant people were free of the constraints (and costs) of online solutions.  All of this he has done for free and in doing so has allowed countless people to expand their horizons.

I use both Forge and Fooocus every day and that is the best way you can reward the guy, to actually use the products he has put out there.

**Comment by SunGreen777** [comment]
@ScionoicS  Delete the topic, I did not want to argue with anyone and did not want to offend anyone. There were questions and they have lost relevance.
